front/src/providers/ApolloAuth0Provider.tsx

```diff
@@ -37,12 +37,19 @@ function ApolloAuth0Provider({ children }: { children: ReactNode }) {
 
   const errorLink = onError(({ graphQLErrors, networkError }) => {
     if (graphQLErrors) {
-      graphQLErrors.map(({ message, locations, path }) =>
+      graphQLErrors.map(({ message, locations, path, extensions }) => {
+        // Supress 'base=0' errors. We are trying to preload queries and cache results.
```

Comment:
two things come to my mind here:
1. we must be careful to not change the format of the error message returned by the BE. In some way, the error message becomes part of the interface that the FE relies on
2. Requesting `base=0` will cause [Sentry reports](https://boxwise.sentry.io/issues/5670060810/?alert_rule_id=12109046&alert_type=issue&notification_uuid=c2ef13f7-4ad3-4c94-8b0d-eb339ae4f9bc&project=6584296&referrer=slack) 

Not sure how far we want to go here but we at least the Sentry errors don't have to necessarily be reported

Reply:
@pylipp Maybe there's a more nuanced and elegant way of going about this, but I'm only skipping this particular message exactly so as not to flare users with unneeded error messages.

About Sentry:
Making a note to suppress this error here: https://trello.com/c/JWPwlcRL/1462-20-fe-update-sentry-config

Reply:
I think I got a better way. Check it out @pylipp !

Reply:
Yeah makes total sense to avoid running the queries with baseId 0 🚀 thx!